js/bitmart.js

```diff
@@ -26,7 +26,7 @@ module.exports = class bitmart extends Exchange {
                 'spot': true,
                 'margin': true,
                 'swap': undefined, // has but unimplemented
-                'future': undefined, // has but unimplemented
+                'future': false,
```

Comment:
@Dan-krm, we still have a lot of references/checks for future markets, should we remove them?

Reply:
> @Dan-krm, we still have a lot of references/checks for future markets, should we remove them?

Yeah there's some old contract code and future references in some of the methods that I can remove if it's no longer working

Reply:
@Dan-krm ok let's do it but in a different PR to avoid blocking this one